### PR TITLE
remove messages screen global states

### LINKED ISSUE #17362: 'Jump to' button is hidden and 'Scroll down' arrow is displayed at the bottom of chat history when opening another chat
# Bug Report

### Reproduction

1. Open chat with some history
2. Scroll up until the `Scroll down` arrow button appears
3. Close the chat
4. Open another chat
5. Check the `Scroll down` and `Jump to` buttons

#### Expected behavior
- The arrow does not appear in the second chat
- `Jump to` is displayed

#### Actual behavior
- The arrow appears even though you're at the bottom of the chat history
- `Jump to` is hidden

https://github.com/status-im/status-mobile/assets/67952253/b58b69f8-ea3a-4034-8257-1294d2109278

### Additional Information

- Status version: nightly 20/09
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/17362

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/messages/list/view.cljs [hunk 1]
```diff
@@ -27,11 +26,7 @@
 (defonce ^:const scroll-animation-input-range [50 125])
 (defonce ^:const min-message-height 32)
 
-(defonce extra-keyboard-height (reagent/atom 0))
 (defonce messages-list-ref (atom nil))
```
Comment (Parveshdhull) [inline src/status_im2/contexts/chat/messages/list/view.cljs:30]: Also tried removing `messages-list-ref` https://github.com/status-im/status-mobile/commit/7c7f670e4de917550b950092368c600659c8758a, but this started to cause weird error https://stackoverflow.com/questions/48045696/flatlist-scrollview-error-on-any-state-change-invariant-violation-changing-on

**Comment by VolodLytvynenko** [comment]
hi @Parveshdhull could you resolve please the existing conflicts in this PR?

**Comment by VolodLytvynenko** [comment]
Hi @Parveshdhull . thank you for PR. No issues from my side. Please let me know if you plan to push any new commits after PR review, otherwise, the PR can be merged :) Thank you!

**Review by ibrkhalil — APPROVED** [review]
This was a much needed and awaited for cleanup
Thanks Parvesh!

File: src/status_im2/contexts/chat/messages/view.cljs [hunk 2]
```diff
@@ -83,9 +82,14 @@
...
         {:insets                            insets
          :scroll-to-bottom-fn               list.view/scroll-to-bottom
-         :show-floating-scroll-down-button? list.view/show-floating-scroll-down-button?}]
+         :show-floating-scroll-down-button? show-floating-scroll-down-button?}]
        [contact-requests.bottom-drawer/view chat-id contact-request-state group-chat])]))
 
 (defn chat
   []
-  [:f> f-chat])
+  (let [reagent-atoms
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/messages/view.cljs]: nitpick: rename to inner-state-atoms, reagent-atoms is too generic imo.

Comment (Parveshdhull) [reply src/status_im2/contexts/chat/messages/view.cljs]: done, thank you

**Pull request merged**